### PR TITLE
Update v3 to v4 how-to re: installer version (442)

### LINKED ISSUE #442: Update v3 to v4 howto re installer version
In our https://rockstor.com/docs/howtos/v3_to_v4.html we have the following:

> Our [downloadable](https://rockstor.com/dls.html) v4 installers have Stable channel release ([4.1.0](https://forum.rockstor.com/t/v4-built-on-opensuse-stable-channel-changelog/8212)) - Jan 2022 pre-installed.

Which is now outdated regarding the version included within our installers.

It is proposed that we modify this text to age better.

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #442.

### This pull request's proposal
 Changes to be committed:
	modified:   v3_to_v4.rst
- update installer reference
- some rewording/extension
- fixed some typos

### Checklist
- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by Hooverdan96** [comment]
linkcheck failure, due to known broken link (addressed pull request #447)
(line   97) broken    https://btrfs.wiki.kernel.org/index.php/Glossary - 404 Client Error: Not Found for url: https://archive.kernel.org/oldwiki/btrfs.wiki.kernel.org/index.php/Glossary.html

File: howtos/v3_to_v4.rst [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
-Rocksor v3 was based on CentOS 7 and is no longer supported.
+Rockstor v3 was based on CentOS 7 and is no longer supported.
 Last updates per channel:
 
 - Stable (3.9.2-57) - April 2020
 - Testing (3.9.1-16) - November 2017
 
-Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers have Stable channel release
-(`4.1.0 <https://forum.rockstor.com/t/v4-built-on-opensuse-stable-channel-changelog/8212>`_) - Jan 2022 pre-installed.
+Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers have the most Stable channel
```
Comment (phillxnet) [inline howtos/v3_to_v4.rst:12]:
```suggestion
Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers aim to have the most recent Stable channel
```

Comment (phillxnet) [reply howtos/v3_to_v4.rst:12]: @Hooverdan96 This fixes the missing 'recent' and gives intent: which we currently are falling short on - and likely will from time to time. I.e. my pending task of rebuilding the installers !!

File: howtos/v3_to_v4.rst [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
+release preinstalled. Below is a non-exhaustive list of notes, recommendations and warnings
+for user planning on going through the upgrade process.
```
Comment (phillxnet) [inline howtos/v3_to_v4.rst:14]:
```suggestion
for those planning on going through the upgrade process.
```

Comment (phillxnet) [reply howtos/v3_to_v4.rst:14]: I like this a little more than 'users' myself or in the typo variant here 'user' ;) .

File: howtos/v3_to_v4.rst [hunk 3]
```diff
@@ -170,19 +176,19 @@ Many bug fixes
...
 that trivially enables highly customised installer creation.
 We also now have ARM64 (e.g. Pi4 / Ten64) compatibility, baring some Rock-ons,
 courtesy of openSUSE's extreme heritage in ARM support.
 
 Also note the following, now we are past the `Jump <https://en.opensuse.org/Portal:Jump>`_ initiative:
 
-- In v3 our upstream of CentOS had in turn it's upstream of RedHat's RHEL.
-- In v4 our upstream of openSUSE has in turn it's increasing binary compatible with SuSE SLES.
+- In v3 our upstream of CentOS had in turn its upstream of RedHat's RHEL.
+- In v4 our upstream of openSUSE has in turn its increasingly binary compatible upstream with SuSE SLES.
```
Comment (phillxnet) [inline howtos/v3_to_v4.rst:188]:
```suggestion
- In v4 our upstream of openSUSE has in turn its binary compatible upstream of SuSE SLES.
```

Comment (phillxnet) [reply howtos/v3_to_v4.rst:188]: Now we are past a Leap 15.3 (Jump) and onto 15.4 I think we can say this: given the Jump initiative is behind us and. With 15.4 being current and our preferred download. Plus they now share repositories as a result.

**Review by phillxnet — CHANGES_REQUESTED** [review]
@Hooverdan96 This is a nice upgrade :). Thanks

I've made a few suggested changes but nothing major. See what you think.
Don't worry about squashing back to a single commit, that is less important here than in our code repositories I think.

File: howtos/v3_to_v4.rst (hunk 1 above)
Comment (Hooverdan96) [reply howtos/v3_to_v4.rst:12]: yes, that sounds good. I will accept that edit.

File: howtos/v3_to_v4.rst (hunk 3 above)
Comment (Hooverdan96) [reply howtos/v3_to_v4.rst:188]: yep, that also makes perfect sense

**Comment by Hooverdan96** [comment]
All changes accepted!

**Pull request merged**